recipes/expat/all/conanfile.py

```diff
@@ -33,11 +34,19 @@ def source(self):
 
     def _configure_cmake(self):
         cmake = CMake(self, parallel=True)
-        cmake.definitions["BUILD_doc"] = "Off"
-        cmake.definitions["BUILD_examples"] =  "Off"
-        cmake.definitions["BUILD_shared"] = self.options.shared
-        cmake.definitions["BUILD_tests"] = "Off"
-        cmake.definitions["BUILD_tools"] = "Off"
+        if Version(self.version) < "2.2.8":
+            cmake.definitions["BUILD_doc"] = "Off"
```

Comment:
I think you can use Pyhon booleans: `True`/`False`

Reply:
You could, but only because they will stringify to "True" "False" and CMake understands those strings too in most contexts. Since these are defined in upstreams's CMakeLists.txt with option(...), the cmake-gui and more IDE integrations would use "Off" and "On" as this recipe (and this patch) already does.

I don't think it really matters either way.